### ISSUE TITLE
How to remove keychain value in a key?

### ISSUE THREAD

**Issue by imycc** [dagostini/DAKeychain#2]
only found shared function in the example.
Are there any way to remove value/object ?

**Comment by dagostini** [comment]
Yep, you can delete the value by setting it to nil, just like a dictionary `keychain["key"] = nil`
